www/blog/postgresql_and_skip_poc.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Building a Real-time Blog with SKIP and PostgreSQL
+description: How to use Skip's reactive data streaming with a PostgreSQL backend
+slug: postgresql_and_skip
+date: 2025-05-16
+authors: hubyrod
+---
+
+Hello Skippers. Today, we are going to plug a [PostgreSQL](https://www.postgresql.org/) service and our starting point is the following project on GitHub: [Skip Postgres Demo](https://github.com/SkipLabs/postrgre_and_skip_poc). We will demonstrate how to use SKIP's reactive data streaming with a PostgreSQL backend. This project implements a simple blog post system with real-time updates using SKIP's streaming capabilities.
+
+![PostgreSQL and SKIP Service](./assets/postgre_and_skip.png)
+
+
+Before we get too far ahead, two things…(1) what you need and (2) what you will learn: 
+
+## What you need: 
+
+- [Node.js](https://nodejs.org/en) (Latest LTS version recommended)
+- [`pnpm`](https://pnpm.io/) package manager
+- [Docker](https://www.docker.com/) (for PostgreSQL)
+- (optional) a Json pretty printer in the terminal, I use [jq](https://jqlang.org/)
+
+## What you will learn:
+
+- Real-time post streaming using SKIP's reactive data system
+- PostgreSQL integration for data persistence
+- RESTful API endpoints for post management
+- Support for streaming data access
+
+## Okay..ready?….let's go!
+
+We are starting from a REST server for the simplest blog post system and we are going to add real-time updates features to it.
+Let's check out the project: 
+```bash
+git clone https://github.com/SkipLabs/postgre_and_skip_poc.git
+cd postgre_and_skip_poc/
+git checkout $(git rev-list --max-parents=0 main)
+```
+
+And yes, you are right we are not starting on HEAD of the main branch, but don't worry we'll get there in no time.
+
+
+Run the `init_and_start_server.sh` script, it will start a PostgreSQL database in a Docker container and fill it in with some data. It will then install and build the whole project with pnpm to finally start it! 
```

Comment:
```suggestion
Run the `init_and_start_server.sh` script: it will start a PostgreSQL database in a Docker container, populate it with some sample data, and then build and start the project with `pnpm`.
```